CONTRIBUTING.md

```diff
@@ -72,10 +78,10 @@ This is a rough outline of how to prepare a contribution:
 
 ### How to build and run the project
 
-Refer to the [Building](https://docs.fluxcd.io/projects/helm-operator/en/stable/contributing/building/) guide to find out how to build from
+Refer to the [Building](https://docs.fluxcd.io/en/stable/contributing/building/) guide to find out how to build from
```

Comment:
The Helm Operator now has its own version of these types of document but they have not ended up in a `stable` release yet. I do however think it would be fine to use `latest` links here, in which case they URL would be: https://docs.fluxcd.io/projects/helm-operator/en/latest/contributing/building/